ivc/src/poseidon_quadri/params.rs

```diff
@@ -0,0 +1,890 @@
+use ark_bn254::Fr as Fp;
+use mina_poseidon::{constants::SpongeConstants, poseidon::ArithmeticSpongeParams};
+use once_cell::sync::Lazy;
+
+// FIXME: move into mina_poseidon when this code is ready to go into production.
+/* Generated by the following command:
+```shell
+sage params.sage --rounds 55 rust 3 bn254 > bn254_poseidon.rs
+```
+with the commit 85cbccd4266cdb567fd47ffc54c3fa52543c2c51
+where the curves have been changed in the script params.sage to use bn254
+*/
+
+use std::str::FromStr;
+
+fn params() -> ArithmeticSpongeParams<Fp> {
```

Comment:
Can't we use Poseidon params from `ivc/src/poseidon/`? It's the same number of rounds, so I'm wondering how are these different?

Reply:
Yes :). Doing it.

Reply:
Done